File: vm_Assignments/vmc/L2.7/fibonacci.c
#include<stdio.h>
int fibonacci(int n)
{
    int x1=1;
    int x2=1;
    printf("%d   %d   ",x1, x2);
    for(int i=0; i<n-2; i++)
    {
        int x3 = x1 + x2;
        printf("%d   ", x3);
        x1 = x2;
        x2 = x3;
    }
}
int main()
{
    fibonacci(10);
}

File: vm_Assignments/vm/Beginning/L2.7/fibonacci.py
def fibonacci(n):
    x1, x2 = 1, 1
    print(x1)
    print(x2)
    for i in range(n-2):
        n = x1 + x2
        x1 = x2
        x2 = n
        print(n)

fibonacci(10)